package.json

```diff
@@ -28,12 +28,16 @@
     "@types/react": "^18.0.17",
     "@types/react-dom": "^18.0.6",
     "@vitejs/plugin-react": "^2.1.0",
-    "babel-jest": "^29.2.2",
     "jest": "^29.2.2",
     "jest-environment-jsdom": "^29.2.2",
+    "jest-svg-transformer": "^1.0.0",
     "jest-transform-stub": "^2.0.0",
     "prettier": "^2.7.1",
     "typescript": "^4.6.4",
-    "vite": "^3.1.0"
+    "vite": "^3.1.0",
+    "vite-plugin-svgr": "^2.2.2"
+  },
+  "optionalDependencies": {
+    "esbuild-linux-64": "^0.15.13"
```

Comment:
necessary?

Reply:
Tried installing globally, didnt work, added it here. it SHOULD only install on linux 64 bit platforms from what I read.